### PR TITLE
Update swagger docs for add and delete apis

### PR DESCRIPTION
**Pull request by sanfern**
Signed-off-by: Santhosh Fernandes <santhosh.fernandes@gmail.com>

### CODE REVIEW

**Review by jniesz — APPROVED** [review]
LGTM

**Review by jaysheth2 — APPROVED** [review]
LGTM

File: docs/docs.go [hunk 1]
```diff
@@ -230,6 +322,23 @@ const docTemplate = `{
                 }
             }
         },
+        "models.L3afBPFProgramNames": {
+            "type": "object",
+            "properties": {
+                "bpf_programs": {
+                    "description": "List of bpf program names to remove",
```
Comment (dthaler) [inline docs/docs.go:329]:
```suggestion
                    "description": "List of eBPF program names to remove",
```

File: docs/swagger.json [hunk 2]
```diff
@@ -222,6 +314,23 @@
                 }
             }
         },
+        "models.L3afBPFProgramNames": {
+            "type": "object",
+            "properties": {
+                "bpf_programs": {
+                    "description": "List of bpf program names to remove",
```
Comment (dthaler) [inline docs/swagger.json:321]:
```suggestion
                    "description": "List of eBPF program names to remove",
```

File: docs/swagger.yaml [hunk 3]
```diff
@@ -101,6 +122,18 @@ definitions:
           $ref: '#/definitions/models.BPFProgram'
         type: array
     type: object
+  models.L3afBPFProgramNames:
+    properties:
+      bpf_programs:
+        $ref: '#/definitions/models.BPFProgramNames'
+        description: List of bpf program names to remove
```
Comment (dthaler) [inline docs/swagger.yaml:129]:
```suggestion
        description: List of eBPF program names to remove
```

File: docs/swagger.json [hunk 4]
```diff
@@ -20,7 +20,69 @@
...
                     "200": {
-                        "description": ""
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/l3af/configs/v1/add": {
+            "post": {
+                "description": "Adding new eBPF Programs on node",
```
Comment (dthaler) [inline docs/swagger.json:30]:
```suggestion
                "description": "Adds new eBPF Programs on node",
```
use grammar consistent with line 13

File: docs/swagger.json [hunk 5]
```diff
@@ -20,7 +20,69 @@
...
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary": "Adding new eBPF Programs on node",
```
Comment (dthaler) [inline docs/swagger.json:37]:
```suggestion
                "summary": "Adds new eBPF Programs on node",
```

File: docs/swagger.json [hunk 6]
```diff
@@ -20,7 +20,69 @@
...
+                "responses": {
+                    "200": {
+                        "description": "OK"
+                    }
+                }
+            }
+        },
+        "/l3af/configs/v1/delete": {
+            "post": {
+                "description": "remove eBPF Programs on node",
```
Comment (dthaler) [inline docs/swagger.json:61]:
```suggestion
                "description": "Removes eBPF Programs on node",
```
Use grammar and capitalization consistent with line 13

File: docs/swagger.json [hunk 7]
```diff
@@ -20,7 +20,69 @@
...
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary": "remove eBPF Programs on node",
```
Comment (dthaler) [inline docs/swagger.json:68]:
```suggestion
                "summary": "Removes eBPF Programs on node",
```

File: docs/swagger.yaml [hunk 8]
```diff
@@ -164,8 +197,48 @@ paths:
...
       responses:
         "200":
-          description: ""
+          description: OK
       summary: Returns details of the configuration of eBPF Programs for a given interface
+  /l3af/configs/v1/add:
+    post:
+      consumes:
+      - application/json
+      description: Adding new eBPF Programs on node
```
Comment (dthaler) [inline docs/swagger.yaml:206]:
```suggestion
      description: Adds new eBPF Programs on node
```

File: docs/swagger.yaml [hunk 9]
```diff
@@ -164,8 +197,48 @@ paths:
...
+        schema:
+          items:
+            $ref: '#/definitions/models.L3afBPFPrograms'
+          type: array
+      produces:
+      - application/json
+      responses:
+        "200":
+          description: OK
+      summary: Adding new eBPF Programs on node
```
Comment (dthaler) [inline docs/swagger.yaml:221]:
```suggestion
      summary: Adds new eBPF Programs on node
```

File: docs/swagger.yaml [hunk 10]
```diff
@@ -164,8 +197,48 @@ paths:
...
+  /l3af/configs/v1/delete:
+    post:
+      consumes:
+      - application/json
+      description: remove eBPF Programs on node
```
Comment (dthaler) [inline docs/swagger.yaml:226]:
```suggestion
      description: Removes eBPF Programs on node
```

File: docs/swagger.yaml [hunk 11]
```diff
@@ -164,8 +197,48 @@ paths:
...
+        schema:
+          items:
+            $ref: '#/definitions/models.L3afBPFProgramNames'
+          type: array
+      produces:
+      - application/json
+      responses:
+        "200":
+          description: OK
+      summary: remove eBPF Programs on node
```
Comment (dthaler) [inline docs/swagger.yaml:241]:
```suggestion
      summary: Removes eBPF Programs on node
```

**Review by dthaler — APPROVED** [review]

**Review by jniesz — APPROVED** [review]

**Pull request merged**